src/build_tools/hooks_hydra.py

```diff
@@ -484,6 +484,17 @@ def pre_module_hook(self, *args, **kwargs):  # pylint: disable=unused-argument
         if self.name == 'ABAQUS':
             self.cfg['modextravars'].update({'ABAQUSLM_LICENSE_FILE': '/apps/brussel/licenses/abaqus/license.lic'})
 
+        # FreeSurfer per-user licenses
+        if self.name == 'FreeSurfer':
+            self.cfg['modluafooter'] = """
+local username = os.getenv("USER") or ""
+local licfile = pathJoin("/apps/brussel/licenses/freesurfer", username .. ".lic")
+if not isFile(licfile) then
+    LmodError("FreeSurfer license file not found for your VSC user. Please contact user support on how to obtain one.")
```

Comment:
mention hpc@vub.be ?

Reply:
fixed in https://github.com/vub-hpc/build_tools/pull/47/commits/b5f8ef77dd93a967a40982bab91b8732b18c9692